phpunit.xml.dist

```diff
@@ -14,6 +14,10 @@
         </testsuite>
     </testsuites>
 
+    <extensions>
+        <bootstrap class="Ergebnis\PHPUnit\SlowTestDetector\Extension"/>
+    </extensions>
```

Comment:
(How) is this used? I see no configuration or annotations…

Reply:
This is the configuration :). And it is included thanks to the `composer.json` requirement.

Here is PHPUnit example output:

```
Detected 2 tests where the duration exceeded the maximum duration.

1. 22.378 (0.500) Doctrine\SqlFormatter\Tests\SqlFormatterTest::testFormatLongConcat
2. 16.231 (0.500) Doctrine\SqlFormatter\Tests\TokenizerTest::testTokenizeLongConcat
```

It helps listing slow tests and in #119 the times will be hugely improved.